invisible_cities/cities/components.py

```diff
@@ -1531,11 +1536,14 @@ def hits_corrector(map_fname : str, apply_temp : bool) -> Callable:
     A function that takes a HitCollection as input and returns
     the same object with modified Ec and Z fields.
     """
-    map_fname = os.path.expandvars(map_fname)
-    maps      = read_maps(map_fname)
-    get_coef  = apply_all_correction(maps, apply_temp = apply_temp, norm_strat = NormStrategy.kr)
-    time_to_Z = (get_df_to_z_converter(maps) if maps.t_evol is not None else
-                 lambda x: x)
+    assert (norm_strat is not NormStrategy.custom) or (norm_value > 0)
```

Comment:
While it doesn't matter as `get_normalization_factor()` doesn't use the `norm_value` provided unless `norm_strat` is custom, should this assert be more stringent (perhaps a XOR rather than OR)? While unlikely, someone may change `get_normalization_factor()` in the future in a manner that makes this logic untenable, such as if a normalisation strategy that isn't custom requires a normalisation value.

Reply:
Agreed. I will also change the assertion to a different exception, this was me being lazy.